site/content/docs/5.3/helpers/icon-link.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: docs
+title: Icon link
+description: Quickly create stylized hyperlinks with Bootstrap Icons or other icons.
+group: helpers
+aliases:
+  - "/docs/icon-link/"
```

Comment:
Do we need this one ? Should we backport it to other pages ?

Reply:
Coming from upstream. No need to backport it IMO but IDK why it is here